nupic/research/frameworks/dynamic_sparse/models/dsnn.py

```diff
@@ -312,8 +318,9 @@ class DSNNMixedHeb(DSNNHeb):
     """Improved results compared to DSNNHeb"""
 
     def _init_coactivation_tracking(self):
-        if self.hebbian_prune_perc is not None:
```

Comment:
As we discussed, can you make it so the parameter can either be a float (and then the same pruning level is applied to all layers) or a list? Also to make it backward compatible, but for simplicity as well. If it is already doing that, please disregard my comment.

Reply:
Already done. There's also a new helper function `SparseModel._make_attr_iterable()` for this purpose.